EnglishTranslation-master/ETC.tsv

```diff
@@ -8120,7 +8120,7 @@ ETC_20150317_008120	If not, the lightning from the Specter Monarch will never en
 ETC_20150317_008121	Tombstone
 ETC_20150317_008122	The Stone Whale is exhausted.
 ETC_20150317_008123	Rajatadpole
-ETC_20150317_008124	local buf = GetBuffByName(self, "Revenge");  local over = GetOver(buf);  return over + 1.. " summons is starting.";
+ETC_20150317_008124	local buf = GetBuffByName(self, "Revenge"); local over = GetOver(buf); return over + 1.. " summons is starting.";
```

Comment:
I would leave this like the original since this is related to developers. Maybe editing this line will have consequences. I don't know.